dali/base/dautils.cpp

```diff
@@ -47,6 +47,59 @@
 #define MIN_REDIRECTION_LOAD_INTERVAL 1000
 
 
+bool isHostInPlane(IPropertyTree *plane, const char *host, bool ipMatch)
+{
+    Owned<IPropertyTreeIterator> hostsIter;
+    if (plane->hasProp("@hostGroup"))
+    {
+        Owned<IPropertyTree> hostGroup = getHostGroup(plane->queryProp("@hostGroup"), true);
+        hostsIter.setown(hostGroup->getElements("*"));
```

Comment:
Should be

`hostsIter.setown(hostGroup->getElements("hosts"));
`

Reply:
presumably equivalent, because only "hosts" there, but I'll chane for clarity.

Reply:
@jakesmith this hasn't been changed yet.

Reply:
@ghalliday - see new commit (what I should have released last time)